scripts/build

```diff
@@ -22,4 +22,4 @@ fi
 echo export GOPATH=$GOPATH
 
 mkdir -p bin
-go build -o bin/host-api
+GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o bin/host-api
```

Comment:
Changed back to the original script, but i'll like to put a note somewhere on the wiki so that people debugging host-api on a MAC should use the cross-compilation commands if the host being used a linux VM.